src/components/home/HomePage.tsx

```diff
@@ -158,7 +157,7 @@ const HomePage: React.FunctionComponent<RouteComponentProps> = ({
                             value={
                                 dataOverview
                                     ? filesize(dataOverview.num_bytes, {
-                                          round: 1
+                                          round: 0
```

Comment:
I disagree with this. I don't think terabytes should be rounded to whole numbers, and we'll be in terabyte territory on the portal fairly soon.

Reply:
agreed

Reply:
how about always show `"{num_bytes/10^9} GB"` ?

Reply:
What about
```javascript
round: num_bytes < 10e12 ? 0 : 1
```

Reply:
okaya